### PR TITLE
Packages reorganization and v2.0.0

### PR DESCRIPTION
**Pull request by SunSince90**
This pull request updates the CNWAN Operator to version 2.0.0 and brings numerous changes, including:

- Packages have been moved and/or reformatted in an effort to be clearer and more organized.
- Better and stronger unit tests
- Better separation of concerns
- Timeouts to all outbound HTTP/S requests, preventing the operator from getting stuck on a request.
- New and clearer structures formats.

To see a full list of changes and descriptions, please consult `CHANGELOG.md` and the commits log.

This pull request fixes and/or addresses #6 and #4.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -85,10 +95,15 @@ func main() {
 	}
 	viper.Set(types.AllowedAnnotationsMap, allowedAnnotations)
 
-	shLog := ctrl.Log.WithName("servicehandler").WithName("gcloudServiceDirectoryHandler")
+	// Create the service registry handler
```
Comment (arnatal) [inline main.go:98]: Maybe we should clarify that we are creating a service registry handler for GCP Service Directory?

**Review by ljakab — COMMENTED** [review]
LGTM.

Really nice work!!!

File: main.go (hunk 1 above)
Comment (SunSince90) [reply main.go]: Just amended. Thank you!

**Comment by arnatal** [comment]
This is great Elis, I really like what you have done with the abstractions of Namespace, Service, and Endpoint. I also like the concept of the Service Registry Broker.

**Pull request merged**